test/cmd/edit.sh

```diff
@@ -6,15 +6,16 @@ set -o pipefail
 
 OS_ROOT=$(dirname "${BASH_SOURCE}")/../..
 source "${OS_ROOT}/hack/util.sh"
+source "${OS_ROOT}/hack/cmd_util.sh"
 os::log::install_errexit
 
 # This test validates the edit command
 
-oc create -f examples/hello-openshift/hello-pod.json
+os::cmd::expect_success 'oc create -f examples/hello-openshift/hello-pod.json'
 
-[ "$(OC_EDITOR=cat oc edit pod/hello-openshift 2>&1 | grep 'Edit cancelled')" ]
```

Comment:
`2>&1` is not necessary any longer, as it was previously used for the `grep` and the helper methods implicitly will check both `stdout` and `stderr` for the desired text.